### DISCUSSION TITLE
Please command examples

### DISCUSSION THREAD

**Discussion by kroryan** [neph1/LlamaTale#17] [General]
Hi!! So i have been trying to use llamatale it is a great idea but i dont know how to use the command and im getting errores all the time 😕 could somebody give me an example of this command ready to use?

tale.cmds.normal.do_say(player: tale.player.Player, parsed: tale.base.ParseResult, ctx: tale.util.Context) → None

If you want to give me some examples of others commands is more than welcome, if i get a example ready to use of one command i think i could figure out what im doing wrong, thanks!!

**Comment by neph1** [comment]
I'll look into it!

**Reply by kroryan** [reply]
Thanks Man, let me know if u update the docs, for me is also fine if you Just send me the command ready to use because if i Just copy paste it like it is and change none for hello i get and syntax error player parameter unknow or something like that, thanks mando and thanks for do this im running this on my phone using termux and Griffin3b i feel like i got my own Aidungeons :)

**Comment by neph1** [comment]
To say something, the command is:
`say name_or_alias: what you want to say`
The thing is, the mud doesn't show the name of the characters, only their description. But in the prancing llama, all characters also have aliases based on their appearance. So, Elid Gald can be adressed as 'gentleman', Shanda Heard as 'fierce woman', etc. It'll be more difficult with the generated characters.

Same goes for moving around. You can usually type the name of the place to move there, so 'main hall' for example. But in the llama, all locations also have compass directions, so, north (n for short), south, east or west. I will try to do this for generated locations, as well.
The 'exits' command will provide some clue to where you can go.

Moreover, I'm not an expert at Tale, but most commands are documented here: https://tale.readthedocs.io/en/latest/api.html#module-tale.cmds
In most cases, remove the 'do_'. You can also type 'help' in game to get some guidelines

I have some doubts about a 3b model being able to produce the necessary JSON outputs, but I'd love to be proven wrong. Let me know how it goes. You could also try a code-llama2 3b model and see if it works better.

Good luck!

**Reply by kroryan** [reply]
Thanks so much Man, well about the 3b models i Just find orcamini and Griffin3b to be able to produce the JSON output but i also change the config so they are no so good and they are a bit shorter but with this 2 models it is working if i try other the output doesnt make sense

**Reply by kroryan** [reply]
Ah, i only find one llama2 3B a wizard vicuna model but it does crazy output,, im not sure if orca mini is llama2 i think not

**Reply by neph1** [reply]
> Thanks so much Man, well about the 3b models i Just find orcamini and Griffin3b to be able to produce the JSON output but i also change the config so they are no so good and they are a bit shorter but with this 2 models it is working if i try other the output doesnt make sense

How has your experience with these models been? What settings did you use to get good results?

**Reply by kroryan** [reply]
well to be honest it was working but was not funny at all sometimes it just gets crazy, what im doing now is using a visionfive2 to run 7b models, i want to create an online web just to do a lite app for only my self to be able to use it anywhere im but its gonna take some time for me to figure out how to do that im totally new on this, if you want to tell me something or give some recomendation im totally open

i mean i can make the Lite app just using the app hermit on my phone what i dont know how to do is to make it online

**Reply by neph1** [reply]
Going online from your local device or from a cloud service? If your local device, it's mostly about firewall configuration. If a cloud service, you need to decide how to serve the LLM. Easiest would be a provider like openai. 
I'm planning to do something similar once the project is stable.

**Reply by kroryan** [reply]
yes, i think i would be much better than AIdungeons, i was trying to configure it to make it online and link it with a domain that i have but i cant open the port 80 for the ipv4 on the router (maybe im crazy i just thouthg  what would be the easy way to do it) well the thing is i cant find any post about how to make a local web to online,

**Reply by kroryan** [reply]
im gonna still looking any way to do it after i have done it i will probably post it around

**Reply by neph1** [reply]
I was planning on doing something like this: https://www.tomshardware.com/how-to/host-public-website-raspberry-pi

**Reply by neph1** [reply]
The thing is, Interactive fiction mode doesn't support multiple users, so it would not be possible to do AIDungeons where everyone can make their own story.

**Reply by kroryan** [reply]
i know but i just want it for me, i would have to implement some login

**Reply by kroryan** [reply]
oh thats perfect thanks so much

**Comment by neph1** [comment]
I just tested, and you can run a story like a MUD (if you specify it in the story). That way you don't need to handle the login system by yourself: https://github.com/neph1/LlamaTale/wiki/Running-a-MUD
